iwyu_include_picker.cc

```diff
@@ -1589,9 +1611,17 @@ IncludeVisibility IncludePicker::ParseVisibility(
 }
 
 IncludeVisibility IncludePicker::GetVisibility(
-    const string& quoted_include) const {
+    const MappedInclude& include) const {
```

Comment:
Let's add an argument here for the default value:

    GetVisibility(const MappedInclude& include,
                       IncludeVisibility default_value = kUnusedVisibility) const;

... so that callers can override the default behavior where it makes sense to be more explicit.

Reply:
Done.